packages/office-ui-fabric-react/src/components/SwatchColorPicker/SwatchColorPicker.Props.ts

```diff
@@ -0,0 +1,114 @@
+import { IButtonProps } from '../../Button';
+
+export interface ISwatchColorPickerProps {
+
+  /**
+   * the number of columns for the swatch color picker
+   */
+  columnCount: number;
+
+  /**
+   * The id for the swatch color picker
+   */
+  id?: string;
+
+  /**
+   * The shape of the color cells, defaults to circle
+   */
+  cellShape?: 'circle' | 'square';
+
+  /**
+   * The id of color cell that is currently selected
+   */
+  selectedId?: string;
+
+  /**
+   * Icon props for the swatch color picker. If given the swatch color picker
+   * will render inside of a menu
+   */
+  swatchColorPickerButtonProps?: IButtonProps;
```

Comment:
should we remove the *swatchColorPicker* prefix from the members interface? They are quite mouthful, the interface is already called ISwatchColorPickerProps so I'd expect something more like menuButton.